### ISSUE TITLE
Autolink settings are lost, TVDB info keeps populating.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dizzydezz/jmm#354]
```
What steps will reproduce the problem?
1. Erase TVDB/themoviedb/tvtract links
2. turn off autolink on all titles
3. wait

What is the expected output? What do you see instead?
wanted no TVDB info as keeps populating wildly incorrect data such as linking 
to F troop, etc

What version of the product are you using? On what operating system?
JMM 3.1.24.0

Please provide any additional information below.


```

Original issue reported on code.google.com by `Nat.asmi...@gmail.com` on 10 Dec 2012 at 9:07

**Comment by GoogleCodeExporter** [comment]
Original comment by `werndly...@gmail.com` on 10 Dec 2012 at 11:16
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
updating anidb data seems to reset autolink to enabled as well
```

Original comment by `Nat.asmi...@gmail.com` on 11 Dec 2012 at 12:50

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.1.25
```

Original comment by `werndly...@gmail.com` on 11 Dec 2012 at 1:47
- Changed state: **Fixed**
